### ISSUE TITLE
why no new GEproton & luxtorpeda  releases?

### ISSUE THREAD

**Issue by ahjolinna** [FlashyReese/decky-wine-cellar#10]
is there a reason why there aren't any new GEproton release after v8-25 and luxtorpeda v67.1.2?

I have prerelease version of Decky and testing version of store channel enabled 

and using Beta-main for SteamOS

**Comment by martpendle** [comment]
Noticed the same issue today as well. The latest proton ge versions do not show in wine cellar but show in protonup-qt

**Comment by FlashyReese** [comment]
Can you all retrieve the following logs for me?
1. Run `sudo journalctl -u plugin_loader.service > ~/deckyloader.log`, this should generate a `deckyloader.log` file in your home folder
2. All log files located in `~/homebrew/logs/decky-wine-cellar`

**Comment by martpendle** [comment]
@FlashyReese would this still be useful if I’m getting now getting a “cannot read property” error?

This error has some more detail for another plugin here but it appears to be the same issue across multiple plugins - https://github.com/mcarlucci/decky-storage-cleaner/issues/9

I’m on the latest preview build of steam os and on the latest client beta

**Comment by FlashyReese** [comment]
@martpendle Nope, I need to update the plugin to use global DFL, so currently it's broken on preview.

**Comment by martpendle** [comment]
@FlashyReese thanks, once it’s updated I’ll grab those logs.

could you let me know more about the DFL update? I want to post it on the Storage cleaner issue. Thank you

**Comment by FlashyReese** [comment]
@martpendle They need to update DFL and [they should implement global DFL to avoid issues like these(Steam Deck Homebrew Discord)](https://discord.com/channels/960281551428522045/960284327445418044/1205840061204144178)

**Comment by martpendle** [comment]
With the latest version I am able to see GE-Proton9-1 so I think this issue is resolved
